test/tests.js

```diff
@@ -17,15 +17,22 @@ describe('OneTrust Forwarder', function() {
             };
         };
 
-    function configureOneTrustForwarderAndInit() {
-        var consentGroups =
-            '[{&quot;jsmap&quot;:null,&quot;map&quot;:&quot;strictly necessary&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;group 1&quot;},\
-            {&quot;jsmap&quot;:null,&quot;map&quot;:&quot;performance&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;group 2&quot;},\
-            {&quot;jsmap&quot;:null,&quot;map&quot;:&quot;functional&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;group 3&quot;},\
-            {&quot;jsmap&quot;:null,&quot;map&quot;:&quot;targeting&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;group 4&quot;},\
-            {&quot;jsmap&quot;:null,&quot;map&quot;:&quot;Test 1&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;userEditable1&quot;},\
-            {&quot;jsmap&quot;:null,&quot;map&quot;:&quot;Test 2&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;userEditable2&quot;},\
-            {&quot;jsmap&quot;:null,&quot;map&quot;:&quot;Test 3&quot;,&quot;maptype&quot;:&quot;ConsentPurposes&quot;,&quot;value&quot;:&quot;group 6&quot;}]';
+    function generateConsentGroupString(mapping) {
+        return '[' + [
+            mapping.map(function(el) {
+                return [
+                    '{' +
+                    // This is usually just `null` so we don't wrap in `&quot`
```

Comment:
Do you know when this is not `null`?

Reply:
No actually, I'm just basing this on things the existing code samples, and that we're not using `jsmap` in our code.